### PR TITLE
New UI in Angular2

### PR DESCRIPTION
**Pull request by szok**
zrzut ekranu 2016-09-07 o 13 00 06

### CODE REVIEW

**Comment by mkurek** [comment]
- Current menu item is not highlighted
- there is no info that report is beign calculated (in background) when downloading csv
- when i go to the costs report, menu in old ui point to the old usages report and monthly costs
- form on monthly costs tab has no background (as it have on usages report view)
- it'll be usefull to show some loading indicator (sth like https://thomas.vanhoutte.be/miniblog/wp-content/uploads/spinningwheel.gif)
- there is no progress bar

**Comment by mkurek** [comment]
- show (partial) table on monthly costs view during recalculation
- show spinner (somewhere) during performing request

File: src/ralph_scrooge/rest/reports.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ def _get_params(self, request):
         return {}
 
     def get(self, request):
+        return_data = request.QUERY_PARAMS.get('return_data', False)
```
Comment (mkurek) [inline src/ralph_scrooge/rest/reports.py]: this isn't used anywhere

Comment (szok) [reply src/ralph_scrooge/rest/reports.py]: my mistake :)

File: src/ralph_scrooge/utils/worker_job.py [hunk 2]
```diff
@@ -66,6 +77,7 @@ def run_on_worker(self, **kwargs):
                     progress = 100
                     cache.delete(key)
         else:
+            print('running new job')
```
Comment (mkurek) [inline src/ralph_scrooge/utils/worker_job.py]: remove it

File: src/ralph_scrooge/utils/worker_job.py [hunk 3]
```diff
@@ -40,18 +40,29 @@ def _clear_cache(cls, **kwargs):
...
             data = self._worker_func(**kwargs)
             return (100, data, {}) if self._return_job_meta else (100, data)
         key = _get_cache_key(self.cache_section, **kwargs)
+        print(key)
         cached = cache.get(key)
         if cached is not None:
             progress, job_id, data = cached
-            connection = django_rq.get_connection(self.queue_name)
-            job = Job.fetch(job_id, connection)
+            print(progress, job_id, data)
```
Comment (mkurek) [inline src/ralph_scrooge/utils/worker_job.py]: remove it

File: src/ralph_scrooge/utils/worker_job.py [hunk 4]
```diff
@@ -40,18 +40,29 @@ def _clear_cache(cls, **kwargs):
...
+        return Job.fetch(job_id, connection)
+
     def run_on_worker(self, **kwargs):
         cache = get_cache(self.cache_name)
         if isinstance(cache, DummyCache):
             # No caching or queues with dummy cache.
             data = self._worker_func(**kwargs)
             return (100, data, {}) if self._return_job_meta else (100, data)
         key = _get_cache_key(self.cache_section, **kwargs)
+        print(key)
```
Comment (mkurek) [inline src/ralph_scrooge/utils/worker_job.py]: remove it

**Comment by mkurek** [comment]
:ok:

**Pull request merged**